zookeeper-server/pom.xml

```diff
@@ -74,13 +74,10 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
-    <dependency>
-      <groupId>ch.qos.logback</groupId>
-      <artifactId>logback-core</artifactId>
-    </dependency>
     <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
+      <scope>test</scope>
```

Comment:
Turn out this is enough to go without `flatten-maven-plugin` as test dependencies are no transitive and skipped by default in owasp dependency-check. 

https://jeremylong.github.io/DependencyCheck/dependency-check-maven/check-mojo.html#skipTestScope